src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/TranscodingReadStream.netcoreapp.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Net.Http.Json
+{
+    internal sealed partial class TranscodingReadStream : Stream
```

Comment:
We shouldn't need a private implementation for .NET 5; you should contribute this as part of https://github.com/dotnet/runtime/issues/30260

Reply:
@GrabYourPitchforks are you actively working on that issue, it is marked for 5.0, is that still the plan? 
I want to assume that these `TranscodingStream`s aren't going to be the fix that closes above issue.

In that case, I think it makes more sense if I close this PR and wait for 30260 to close.